### DISCUSSION TITLE
No module name baseline enjoy_kuka_diverse_object_grasping

### DISCUSSION THREAD

**Discussion by AndreaIncertiDelmonte** [bulletphysics/bullet3#3794] [General]
Hi,
I was trying to run the demo showed in this video https://www.youtube.com/watch?v=y7scmgsGQdA

Inside a python 3.5 virtual env I ran the following commands:
```
pip3 install pybullet tensorflow gym
python -m pybullet_envs.baselines.enjoy_kuka_diverse_object_grasping
```
and I got this import error:
```
bullet3/examples/pybullet/gym/pybullet_envs/baselines/enjoy_kuka_grasping.py
from baseline import deepq
ImportError: No module name 'baselines'
```
I also tried to install pybulled from source same results.

It seems the module deepq is missing.

Thnks
Andrea

**Comment by erwincoumans** [comment]
That baselines line shouldn't be there. For the time being, try use pip install baselines?
